Examples/PyNetKet/ground_state_iter.py

```diff
@@ -17,50 +18,48 @@
 ha = nk.operator.Ising(h=1.0, hilbert=hi)
 
 # Machine
-ma = nk.machine.RbmSpinSymm(hilbert=hi, alpha=4)
+ma = nk.machine.RbmSpinV2(hilbert=hi, alpha=2)
 ma.init_random_parameters(seed=SEED, sigma=0.01)
 
 # Sampler
-sa = nk.sampler.MetropolisLocal(machine=ma)
+sa = nk.sampler.MetropolisLocalV2(machine=ma, batch_size=32)
 sa.seed(SEED)
 
-# Optimizer
-op = nk.optimizer.Sgd(learning_rate=0.1)
-
 mpi_rank = nk.MPI.rank()
 
 if mpi_rank == 0:
     e0 = -1.27455 * N
+    # e0 = nk.exact.lanczos_ed(ha).eigenvalues[0]
     print("  E0 = {: 10.4f}".format(e0))
 
 FORMAT_STRING = (
-    "{: 4} | {: 10.4f} ± {:.4f} | {:9.4f} ± {:.4f} | {:.4f} | {: 7} | {:.2e} | {:.2e}"
+    "{: 4} | {: 10.4f} ± {:.4f} | {:9.4f} | {:.4f} | {:.4f} | {: 7} | {:.2e} | {:.2e}"
 )
-HEADER_STRING = "step | E          ± σ(E)   | (ΔE)^2    ± σ(...) | Accept | rank(S) | λ_min(S) | λ_max(S)"
+HEADER_STRING = "step | E          ± σ(E)   | (ΔE)^2    | Rhat   | Accept | rank(S) | λ_min(S) | λ_max(S)"
 
 
 def output(vmc, step):
     obs = vmc.get_observable_stats()
     if mpi_rank == 0:
-        energy = obs["Energy"]["Mean"]
-        sigma = obs["Energy"]["Sigma"]
-        variance = obs["EnergyVariance"]["Mean"]
-        vsigma = obs["EnergyVariance"]["Sigma"]
+        energy = obs["Energy"].mean
+        sigma = obs["Energy"].error_of_mean
+        variance = obs["Energy"].variance
+        rhat = obs["Energy"].R
 
-        S = vmc.last_S_matrix
-        w = eigvalsh(S)
+        # S = vmc._sr.last_covariance_matrix
+        # w = eigvalsh(S)
 
         print(
             FORMAT_STRING.format(
                 step,
-                energy,
+                energy.real,
                 sigma,
                 variance,
-                vsigma,
-                sa.acceptance[0],
-                vmc.last_rank,
-                w.min(),
-                w.max(),
+                rhat,
+                1,  # sa.acceptance,
```

Comment:
MetropolisLocalV2 doesn't have acceptance?

Reply:
Right now it doesn't; that still needs to be implemented. (@twesterhout)

Reply:
Ok, see #278.